event.go

```diff
@@ -41,3 +41,39 @@ func (n *nullEvent) Type() string {
 
 // Sanity check
 var _ Event = (*nullEvent)(nil)
+
+type jsonPatchEvent struct {
```

Comment:
nice, guessing this would implement an interface in core

Reply:
It actually implements the `Event` interface in this same file. But not sure yet about what that interface means... maybe we'll have it more clear when plumbing with `go-textile-threads`

Reply:
sgtm